### PR TITLE
#170602242 Design landing page

### PR DESCRIPTION
**Pull request by nziokaivy**
#### What does this PR do?

Add the landing page of the application

#### Description of Task to be completed?

- Add landing page HTML layout
- Add landing page style
- Style landing page responsive mobile design

#### How should this be manually tested?

- Clone this repository using the command `git clone https://github.com/andela/team-odd-bn-frontend.git`
- Then navigate into the project `cd team-odd-bn-frontend`
- Switch to the landing page branch using the command: `git checkout ch-landing-page-170602242`
- Run the command `npm i `to install all the packages
- Run `npm run dev` to run the server which will land you to the landing page

#### Any background context you want to provide?

As a user, I would like to view the landing page that can assist in interacting with the application

#### What are the relevant pivotal tracker stories?
[#170602242](https://www.pivotaltracker.com/story/show/170602242)
###### Desktop view

image

###### Mobile view

image

### CODE REVIEW

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Hi @nziokaivy , Correct the responsiveness in ipad and ipad pro then we will be good to go.

Screen Shot 2020-01-09 at 18 37 09

**Review by NiyongaboEric — APPROVED** [review]
Well done @nziokaivy!

File: src/components/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../assets/css/style.css';
+import logo from '../assets/images/logo.png';
+
+const landingPage = () => (
+  <div className="row">
+    <div className="column-left" style={{ backgroundColor: 'white' }}>
+      <div className="main-logo">
```
Comment (victkarangwa) [inline src/components/LandingPage.js]: @nziokaivy 
Is it a good idea to have inline styles?
Maybe we can place this style: `backgroundColor: 'white'` in external styles

**Review by victkarangwa — COMMENTED** [review]
Good job @nziokaivy 
Just attend to the feedback I left

**Review by hezronkimutai — APPROVED** [review]

**Review by victkarangwa — APPROVED** [review]
Good job @nziokaivy !

**Review by shemaeric — APPROVED** [review]
Good work @nziokaivy

**Pull request merged**